docs/otlp/README.md

```diff
@@ -0,0 +1,39 @@
+# OpenTelemetry Protocol (OTLP)
+
+The [OpenTelemetry
+Specification](https://github.com/open-telemetry/opentelemetry-specification)
+defines the [OpenTelemetry Protocol
+(OTLP)](https://github.com/open-telemetry/opentelemetry-proto/tree/main/docs)
+which is a standard for exporting telemetry in a vendor agnostic way.
+OpenTelemetry .NET ships
+[OpenTelemetry.Exporter.OpenTelemetryProtocol](../../src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md)
+which is an implementation of OTLP and may be used to export telemetry to any
+backend with OTLP support or to the [OpenTelemetry
+Collector](https://github.com/open-telemetry/opentelemetry-collector).
+
+## Getting started
+
+If you are new to the [OpenTelemetry Protocol
+(OTLP)](https://github.com/open-telemetry/opentelemetry-proto/tree/main/docs),
+it is recommended to first follow the [getting started in 5 minutes - ASP.NET
+Core Application](./getting-started-aspnetcore/README.md) guide or the [getting
+started in 5 minutes - Console Application](./getting-started-console/README.md)
+guide to get up and running.
```

Comment:
could we make it clear that the linked tutorials are to get familiariaty with OTel .NET itself, and they export telemetry to console for learning purposes. And this doc shows exporting in the OTel's standard -the OTLP exporter..

Reply:
After checking existing docs, this feels like we are duplicating a lot of content from getting started.. I don't know the best way to address that yet..

Reply:
To address my own concern about duplicating the content from getting started:

Wondering if it is better to move this under OTLP Exporter readme instead? It won't be a full blown example (with .csproj etc.), but .md file which shows the expected content of program.cs and .csproj along with instruction to use Aspire to view the telemetry. The current one points to example in this repo, which is setting up otel collector, so this can be a good replacement.

https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol#otlp-exporter-for-opentelemetry-net